portal/views/assessment_engine.py

```diff
@@ -42,6 +42,7 @@
 from ..models.role import ROLE
 from ..models.user import User, current_user, get_user_or_abort
 from .portal import check_int
+from ..trace import dump_trace, establish_trace
```

Comment:
I think this import needs to be in the line above for the block to remain alphabetically sorted (`".."` < `".t"`)